core/unit_test/CMakeLists.txt

```diff
@@ -267,7 +274,66 @@ if(Kokkos_ENABLE_OPENMPTARGET)
     ${CMAKE_CURRENT_BINARY_DIR}/openmptarget/TestOpenMPTarget_ViewMapping_subview.cpp
     ${CMAKE_CURRENT_BINARY_DIR}/openmptarget/TestOpenMPTarget_ViewOfClass.cpp
     ${CMAKE_CURRENT_BINARY_DIR}/openmptarget/TestOpenMPTarget_WorkGraph.cpp
-  )
+    )
+endif()
+
+## FIXME_OPENMPTARGET - Comment non-passing tests with the nvidia compiler
```

Comment:
```suggestion
## FIXME_OPENMPTARGET - Comment non-passing tests with the PGI compiler
```

Reply:
Well even though the compiler_id is PGI, it is technically `nvidia` now and hence I named it thus. Also I should probably change the check from `PGI` to `NVHPC`. That's the preferred way of calling the nvidia compiler.

Reply:
Sure. I'm not opposed to that change but it would be nice if they are consistent. :slightly_smiling_face: For me, `nvidia` compiler means `nvcc` and I guess others have the same confusion.

Reply:
Yes, I understand. Will update the `KOKKOS_COMPILER_ID` to use `NVHPC` and update the comments accordingly.

Reply:
Looks like the `NVHPC` compiler-id is only available since cmake/3.20 
https://cmake.org/cmake/help/v3.20/variable/CMAKE_LANG_COMPILER_ID.html#variable:CMAKE_<LANG>_COMPILER_ID

Reply:
You can always rename `PGI` to `NVHPC` when setting `KOKKOS_CXX_COMPILER_ID` according to `CMAKE_CXX_COMPILER_ID`.

Reply:
Actually, I realized that the `ifdef` is not needed there anymore. So deleted that code.

Reply:
Edited the comments in CMakeLists too.